### DISCUSSION TITLE
Unable to check Sunbird RC APIs For Certificate Create Download and Search link

### DISCUSSION THREAD

**Discussion by yogeshsarmalkar-trigyn** [Sunbird-Lern/Community#140] [Documentation]
Hi Team,

I am unable to access the "Sunbird RC APIs For Certificate Create Download and Search" link mentioned in the below screenshot, 
![image](https://github.com/Sunbird-Lern/Community/assets/77775434/444e54f0-5e7f-401e-b12d-586610888c7a)

I am getting access issue to check that documentation when clicking on that link.
![image](https://github.com/Sunbird-Lern/Community/assets/77775434/aed2624e-47b1-42eb-85fc-7d43f7b44577)

Please help with the access or provide an alternative !!!

**Comment by AmiableAnil** [comment]
@chitranshu-keshav can you help here?

**Comment by yogeshsarmalkar-trigyn** [comment]
@AmiableAnil Now I am able to access....It worked when I clicked on sign in button.

**Reply by Jayaprakash8887** [reply]
@yogeshsarmalkar-trigyn Please mark this thread as resolved
